bin/git-forgit

```diff
@@ -525,24 +525,28 @@ _forgit_reset_head() {
 
 _forgit_stash_show_preview() {
     local stash
-    stash=$(echo "$1" | cut -d: -f1)
-    _forgit_git_stash_show "$stash" | _forgit_pager diff
+    stash=$(echo "$1" | _forgit_extract_stash_name)
+    shift
+    echo "$stash" | xargs -I% git show --color=always -U"$_forgit_preview_context" % -- "$@" | _forgit_pager show
```

Comment:
Is `xargs` really needed here? Seems to be working fine without it... From what I can see, there will always be only a single argument passed to this function, so `-- "$@"` would not do anything or am I missing something?
This version seems to have the same behavior.
```bash
 show --color=always -U"$_forgit_preview_context" "$stash" | _forgit_pager show
```

Reply:
Okay, there were actually some more places where we used this random `xargs` expression. I cleaned up all of them and rewrote them without `xargs`.
@sandr01d Can you have a look? Should we extract this into a dedicated PR?

Reply:
Thanks for the update. I think you might have missed my comment regarding the function arguments that are passed to `git show` with `-- "$@"`. I think this is not need either.
I'm fine with keeping the cleanup in this PR, as long as we don't squash the commits.